dashboards/MySQL_Replication.json

```diff
@@ -266,7 +266,7 @@
                 "rgba(36, 112, 33, 0.97)"
             ],
             "datasource": "Prometheus",
-            "description": "Displays whether the SQL Thread is in the running state or not. Applies only to a Slave host. The SQL Thread reads the events from the local relay log file and applies them to the Slave host.  Depending on the format of the binary log it can read query statements (STATEMENT format) and re-executes them, or by reading row changes (ROW format) and applying only the changes.\n\nPossibile values:\n\n**Yes** – The SQL Thread is running and is applying events from the realy log to the local Slave host\n\n**No** – The SQL Thread is not running because it is not yet started, or because of an error\n\nSee also:",
```

Comment:
Could you also replace "realy" by "relay"?

Reply:
b499619 :+1: